src/sentry/integrations/gitlab/client.py

```diff
@@ -134,31 +135,46 @@ def get_user(self):
         """
         return self.get(GitLabApiClientPath.user)
 
-    def search_group_projects(self, group, query=None, simple=True):
-        """Get projects for a group
+    def search_projects(self, group=None, query=None, simple=True):
+        """Get projects
 
         See https://docs.gitlab.com/ee/api/groups.html#list-a-group-s-projects
+        and https://docs.gitlab.com/ee/api/projects.html#list-all-projects
         """
 
         def gen_params(page_number, page_size):
-            # simple param returns limited fields for the project.
+            # Simple param returns limited fields for the project.
             # Really useful, because we often don't need most of the project information
-            return {
+            params = {
                 "search": query,
                 "simple": simple,
-                "include_subgroups": self.metadata.get("include_subgroups", False),
+                "order_by": "last_activity_at",
                 "page": page_number + 1,  # page starts at 1
                 "per_page": page_size,
             }
+            if group:
+                params.update(
+                    {
+                        "include_subgroups": self.metadata.get("include_subgroups", False),
+                    }
+                )
+            return params
 
         def get_results(resp):
             return resp
 
-        return self.get_with_pagination(
-            GitLabApiClientPath.group_projects.format(group=group),
-            gen_params=gen_params,
-            get_results=get_results,
-        )
+        if group:
+            return self.get_with_pagination(
+                GitLabApiClientPath.group_projects.format(group=group),
+                gen_params=gen_params,
+                get_results=get_results,
+            )
+        else:
+            return self.get_with_pagination(
+                GitLabApiClientPath.projects,
```

Comment:
I added this conditional because I re-read the original PR ([here](https://github.com/getsentry/sentry/pull/15188)) and also saw that the `/projects` path was added but wasn't used at all - obviously I wasn't the original author but I _think_ this is what was intended.